libs/block-editor/src/lib/demo/block-editor.component.ts

```diff
@@ -41,16 +47,20 @@ export class BlockEditorComponent implements OnInit {
                     element: document.querySelector('#bubbleMenu'),
                     tippyOptions: {
                         maxWidth: 'none',
-                        placement: 'bottom-start',
+                        placement: 'top-start',
                         trigger: 'manual'
                     }
                 }),
                 // Marks Extensions
                 Underline,
-                TextAlign.configure({
-                    types: ['heading', 'paragraph'],
-                })
+                TextAlign.configure({ types: ['heading', 'paragraph', 'listItem'] }),
+                Highlight.configure({ HTMLAttributes: { class: 'highlighted'} }),
+                Link.configure({ openOnClick: true })
             ]
         });
     }
+
+    openForm() {
+        this.linkForm.showForm();
+    }
```

Comment:
Like my previous comment, let's load this dynamically, and hopefully, we can have a nice fade-in animation.